client/js/components/assessments/item.jsx

```diff
@@ -7,211 +7,124 @@ import UniversalInput         from "./universal_input";
 export default class Item extends React.Component{
 
   static propTypes = {
-    question         : React.PropTypes.object.isRequired,
-    currentItemIndex : React.PropTypes.number.isRequired,
-    questionCount    : React.PropTypes.number.isRequired,
-    messageIndex     : React.PropTypes.number.isRequired,
-    confidenceLevels : React.PropTypes.bool.isRequired,
-    outcomes         : React.PropTypes.object
+    question                   : React.PropTypes.object.isRequired,
+    currentItemIndex           : React.PropTypes.number.isRequired,
+    questionCount              : React.PropTypes.number.isRequired,
+    messageIndex               : React.PropTypes.number.isRequired,
+    confidenceLevels           : React.PropTypes.bool.isRequired,
+    nextQuestion               : React.PropTypes.func.isRequired,
+    prevQuestion               : React.PropTypes.func.isRequired,
+    confidenceLevel            : React.PropTypes.func.isRequired,
```

Comment:
Please add a TODO comment to rename `confidenceLevel` to something more explanatory.